examples/dbmgr.py

```diff
@@ -162,3 +169,36 @@ def removeDB(self):
         widget.deleteLater()
         # send the database list and a logging message
         self.sendDB(False, db.name)
+
+    @pyqtSlot()
+    def openCloseDatacalc(self):
+        """Opens or closes a datacalc app.
+
+        If the dataclac app is open, close it.
+        Otherwise, open a new datacalc app.
+        """
+        if self.openCloseDatacalcResult:
+            if not self.openCloseDatacalcResult.done:
+                print("DBMgrApp.openCloseDatacalc(): The previous swiftcall must be done.")
+                return
+            if self.openCloseDatacalcResult.success:
+                self.isDatacalcOpen = not self.isDatacalcOpen
+        if self.isDatacalcOpen:
+            self.openCloseDatacalcResult = self.swiftcall.destroyApp(name="datacalc")
```

Comment:
Maybe the button text should be changed.

Reply:
Wow.. I completely forgot🥲 I will change it.

Reply:
Umm.. I need to track the result to set the text to show exactly, but I didn't want to implement it because I was only trying to show a simple example. Can I just show 'open or close datacalc'? @kangz12345